tests/testValueProviders.py

```diff
@@ -235,8 +243,13 @@ def checkOutput():
         for t in threads:
             t.join()
 
+
+        numpy.testing.assert_equal(opCompute.executionCount, 1)
```

Comment:
As the CI testing has just revealed, this assertion seems occasionally fail.  I'm not sure that it's correct.  In the current implementation of `OpValueCache`, it is possible for the value to be requested more than once if multiple requests are created and cancelled very quickly.  I don't think that's a problem, as long as the requests that *weren't* cancelled eventually get a result.  I'm going to remove this assert.  If you think that's wrong, let me know.